diagnostic_updater/diagnostic_updater/_diagnostic_updater.py

```diff
@@ -229,40 +229,22 @@ class Updater(DiagnosticTaskVector):
     reason.
```

Comment:
If force_update is dropped, documentation block needs to be updated, as it still mentions this feature.